pkg/hub/manager/manager.go

```diff
@@ -95,11 +95,18 @@ func Start(meshClient client.Client, ctx context.Context) {
 	)
 
 	// create slice-controller recorder
-	spokeSliceEventRecorder := events.NewEventRecorder(mgr.GetEventRecorderFor("spokeSlice-controller"))
+	spokeSliceEventRecorder := mevents.NewEventRecorder(meshClient, mgr.GetScheme(), ossEvents.EventsMap, mevents.EventRecorderOptions{
+		Cluster:   ClusterName,
+		Project:   ProjectNamespace,
+		Component: "spokeSlice-controller",
```

Comment:
We can't have event with	Component: "spokeSlice-controller",
as it is exposed to externally

Reply:
okay, will change "spoke" to "worker" across this change